src/commonMain/kotlin/exchange.dydx.abacus/calculator/V2/AccountCalculatorV2.kt

```diff
@@ -60,19 +72,138 @@ internal class AccountCalculatorV2(
                     ?: subaccounts[parentSubaccountNumber]
                     ?: InternalSubaccountState(subaccountNumber = parentSubaccountNumber)
 
-                // TODO: Add other merges
+                parentSubaccount = mergeChildOpenPositions(
+                    parentSubaccount = parentSubaccount,
+                    childSubaccountNumber = subaccountNumber,
+                    childSubaccount = subaccount,
+                )
+
+                parentSubaccount = mergeChildPendingPositions(
+                    parentSubaccount = parentSubaccount,
+                    childSubaccount = subaccount,
+                    markets = marketsSummary.markets,
+                )
 
                 parentSubaccount = mergeOrders(
                     parentSubaccount = parentSubaccount,
                     childSubaccount = subaccount,
                 )
+
+                parentSubaccount = sumEquity(
+                    parentSubaccount = parentSubaccount,
+                    childSubaccount = subaccount,
+                )
+
                 groupedSubaccounts[parentSubaccountNumber] = parentSubaccount
             }
         }
         account.groupedSubaccounts = groupedSubaccounts
         return account
     }
 
+    private fun mergeChildOpenPositions(
+        parentSubaccount: InternalSubaccountState,
+        childSubaccountNumber: Int,
+        childSubaccount: InternalSubaccountState,
+    ): InternalSubaccountState {
+        val parentOpenPositions = parentSubaccount.openPositions
+        val modifiedOpenPositions = parentOpenPositions?.toMutableMap() ?: mutableMapOf()
+        val childOpenPositions = childSubaccount.openPositions
+        for ((market, childOpenPosition) in childOpenPositions ?: emptyMap()) {
+//            modifiedChildOpenPosition?.safeSet(
```

Comment:
I think this is referenced in the margin calculator... will add it if necessary when we convert the margin calculator